plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveMetadata.java

```diff
@@ -73,6 +74,25 @@ public void testCreatePredicate()
                 false));
     }
 
+    @Test
+    public void testCreatePredicateWithEmptyPartition()
+    {
+        ImmutableList.Builder<HivePartition> partitions = ImmutableList.builder();
+        Domain domain = createPredicate(ImmutableList.of(TEST_COLUMN_HANDLE), partitions.build())
```

Comment:
Use `ImmutableList.of()` instead of defining extra field `partitions` for keeping the code smaller.

Reply:
fixed